README.md

```diff
@@ -1,106 +1,112 @@
+[![github.io]](https://itisfoundation.github.io/)
+[![itis.dockerhub]](https://hub.docker.com/u/itisfoundation)
+
+
+<!-- THIS SERVICE MALFUNCTIONS
+[![webserver]](https://microbadger.com/images/itisfoundation/webserver "More on itisfoundation/webserver:staging-latest image")
+[![director]](https://microbadger.com/images/itisfoundation/director "More on itisfoundation/director:staging-latest image")
+[![sidecar]](https://microbadger.com/images/itisfoundation/sidecar "More on itisfoundation/sidecar:staging-latest image")
+[![storage]](https://microbadger.com/images/itisfoundation/storage "More on itisfoundation/storage:staging-latest image")
+-->
+
+
+<!-- ADD HERE ALL BADGE URLS -->
+[Requires.io]:https://img.shields.io/requires/github/ITISFoundation/osparc-simcore.svg
+[travis-ci]:https://travis-ci.org/ITISFoundation/osparc-simcore.svg?branch=master
+[coverals.io]:https://coveralls.io/repos/github/ITISFoundation/osparc-simcore/badge.svg?branch=master
+[github.io]:https://img.shields.io/website-up-down-green-red/https/itisfoundation.github.io.svg?label=documentation
+[itis.dockerhub]:https://img.shields.io/website/https/hub.docker.com/u/itisfoundation.svg?down_color=red&label=dockerhub%20repos&up_color=green
+[webserver]:https://img.shields.io/microbadger/image-size/itisfoundation/webserver/staging-latest.svg?label=webserver&style=flat
+[director]:https://img.shields.io/microbadger/image-size/itisfoundation/director/staging-latest.svg?label=director&style=flat
+[sidecar]:https://img.shields.io/microbadger/image-size/itisfoundation/sidecar/staging-latest.svg?label=sidecar&style=flat
+[storage]:https://img.shields.io/microbadger/image-size/itisfoundation/storage/staging-latest.svg?label=storage&style=flat
+<!---------------------------->
+
+## Overview
+
+![service-web](docs/img/service-interaction.svg)
+
+
+A graph view of services, ports, volumes and service dependencies:
+![](docs/img/docker-compose-wo-networks.png)
+
+## Contributing
+
+Would you like to make a change or add something new? Please read the [contributing guidelines](CONTRIBUTING.md).
+## Usage
+
+```bash
+  # clone repo
+  git clone https://github.com/ITISFoundation/osparc-simcore.git
+  cd osparc-simcore
+
+  # show setup info and build core services
+  make info build
+
+  # starts swarm and deploys services
+  make up-prod
+
+  # display swarm configuration
+  make info-swarm
+
+  # open browser in:
+  #  localhost:9081 - simcore front-end site
+  #
+  xdg-open http://localhost:9081/
+
+  # stops
+  make down
+```
+
+## Requirements
+
+To build and run:
+
+- docker
+- make >=4.2
+- awk, jq (optional tools within makefiles)
+
+To develop, in addition:
+
+- python 3.6 (this dependency will be deprecated soon)
+- nodejs for client part (this dependency will be deprecated soon)
+- swagger-cli (make sure to have a recent version of nodejs)
+- on MacOS you might need to [replace your MacOS utilities with GNU utils](https://apple.stackexchange.com/a/69332) 
```

Comment:
since you are changing this, I recall somebody got confused with our instructions regarding OS? was it you?

Please make sure that it is clear that
a) our system is intended to be developed mainly UNDER LINUX (Ubuntu recommended)
b) it can potentially be developed/deployed in macos and windows with some tricks but support IS NOT GUARANTEED

My advice is that you add a subsection that includes b). Perhaps replace ``##### Setup for **windows**`` by ``##### Setup in other OS?`` and add there the comments about MacOS and windows